.golangci.yml

```diff
@@ -0,0 +1,88 @@
+linters:
+  enable:
```

Comment:
[not-blocking] the golangci team sometimes recommends to disable all (`disable-all: true`) to prevent new versions of the linter introducing unexpected new linters but up to you if you want to disable all and manually enable what you need (which you are kind of already doing)

Reply:
Good catch, I'll add that.